### PR TITLE
feat: donut inference code, test and sample receipt

### PR DESCRIPTION
**Pull request by LaverdeS**
- Added [`donut.py`](https://github.com/Unstructured-IO/unstructured-inference/blob/d6226418d79268b31cf6f247809eef6e1827a0fb/unstructured_inference/models/donut.py): code to run donut inference via constructing an `UnstructuredDonutModel()`
- Added sample receipt to `sample-docs/` for testing
- Added tests with 100% coverage for new [`donut.py`](https://github.com/Unstructured-IO/unstructured-inference/blob/d6226418d79268b31cf6f247809eef6e1827a0fb/unstructured_inference/models/donut.py)

Here is short snippet showing how to use the new script:
```
from unstructured_inference.models import donut
from PIL import Image

model_path = "unstructuredio/donut-base-sroie"
processor_path = "unstructuredio/donut-base-sroie"
config_path = "unstructuredio/donut-base-sroie"

donut_model = donut.UnstructuredDonutModel()
donut_model.initialize(
        model=model_path, processor=processor_path, config=config_path, task_prompt="<s>"
    )
image_path = "./sample-docs/receipt-sample.jpg"
with Image.open(image_path) as image:
    prediction = donut_model.predict(image)

print(f"Prediction:")
for k, v in prediction.items():
    print("{}: {}".format(k, v))
```

### CODE REVIEW

File: test_unstructured_inference/models/test_donut.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+from transformers import DonutSwinModel
+from PIL import Image
+
+
+@pytest.mark.parametrize(
+    "model_path, processor_path, config_path",
+    [
+        ("crispy_donut_path", "crispy_proc", "crispy_config"),
+        ("cherry_donut_path", "cherry_proc", "cherry_config"),
```
Comment (MthwRobinson) [inline test_unstructured_inference/models/test_donut.py:12]: No sprinkle donuts?

**Review by MthwRobinson — COMMENTED** [review]
Implementation looks great. Could you add a summary and test instructions to the PR description so we can try it out? Looks good otherwise.

As a side note, I don't think we ever cut a release for `0.2.5`

**Comment by LaverdeS** [comment]
> Regarding https://github.com/Unstructured-IO/unstructured-inference/pull/44#pullrequestreview-1283074004

Oh OK, Shall I go back to 0.2.5 and update the CHANGELOG.md accordingly?

**Comment by MthwRobinson** [comment]
No you're good, I'll do a release for `0.2.5` later today. You can keep yours `0.2.6`

File: test_unstructured_inference/models/test_donut.py (hunk 1 above)
Comment (LaverdeS) [reply test_unstructured_inference/models/test_donut.py:12]: sprinkle_donut name is reserved for the best model version 😁 I promise. We are still learning to bake them perfectly 🤤

**Review by MthwRobinson — APPROVED** [review]
All of my comments have been addressed. LGTM from my end.

**Pull request merged**